### PR TITLE
Adding repository tag and updating paths for full and partial matches

### PR DESCRIPTION
**Pull request by edisinovcic**
Solves: https://github.com/ethereum/sourcify/issues/181
Affects: https://github.com/Shard-Labs/remix-sourcify/issues/29

### CODE REVIEW

**Comment by edisinovcic** [comment]
Add version of 1 in every tag for now.

**Comment by edisinovcic** [comment]
Name should be manifest

File: src/chains.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "network": "mainnet",
       "networkId": 1,
       "nativeCurrency": {"name":"Ether","symbol":"ETH","decimals":18},
-      "web3": ["https://mainnet.infura.io/v3/891fe57328084fcca24912b662ad101f"],
```
Comment (ligi) [inline src/chains.json:22]: when we just change the api keys here - what will prevent the same thing happens again?

Comment (edisinovcic) [reply src/chains.json:22]: It will be moved to .env. Still the tests are not working properly so I have to fix them first. :)

**Comment by edisinovcic** [comment]
{
  timestamp: 120000,
  version: 1.0.0
}

**Comment by edisinovcic** [comment]
Solved here: https://github.com/ethereum/sourcify/pull/193

**Pull request closed without merging**